### PR TITLE
Fix for the bug DOCTEAM-811

### PR DESCRIPTION
**Pull request by sounix000**
### PR creator: Description

Fixes the documentation bug [DOCTEAM-811](https://jira.suse.com/browse/DOCTEAM-811). The detailed findings behind the resolution is available in [this comment](https://jira.suse.com/browse/DOCTEAM-811?focusedCommentId=1266857&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1266857) and a supplemental screenshot is available in the comment below it.

**Note to reviewer**: For actual content change relevant for the bugfix, see the changes in the *lines 281-293*. The rest of the changes that you see are all XML formatting done by `daps`.

### PR creator: Are there any relevant issues/feature requests?

* [bsc#1205328](https://bugzilla.suse.com/show_bug.cgi?id=1205328)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

**Note for merge**: Cherry-picks of this commit for older maintenance branches:
- `maintenance/SLE15SP4`
  - [Cherry-pick of primary fix, XML formatting, and image optimization](https://github.com/SUSE/doc-sle/commit/0016180104e8c8c7e94010e321cb3e781ec27f88)
  - [Editorial fixes](https://github.com/SUSE/doc-sle/commit/7a61bf2aed849941b0887c66a7d24ba862f931f9)
- `maintenance/SLE15SP3`
  - [Cherry-pick of primary fix, XML formatting, and image optimization](https://github.com/SUSE/doc-sle/commit/fbc0bdd9ab1f253162fdca9fe7651b21088a4f0f)
  - [Editorial fixes](https://github.com/SUSE/doc-sle/commit/66f93e36c4a76bdaace3bcb973bc1b6aa1347811)
- `maintenance/SLE15SP2`
  - [Cherry-pick of primary fix, XML formatting, and image optimization](https://github.com/SUSE/doc-sle/commit/97ecbbcc1bab83e584b173a54d7760020ead9785)
  - [Editorial fixes](https://github.com/SUSE/doc-sle/commit/af307374e673cd8e8ac87cbbe508a92c94f27f4b)

Based on [this Slack conversation](https://suse.slack.com/archives/C02DR0BQ9C0/p1685025468701189?thread_ts=1685024026.480899&cid=C02DR0BQ9C0), I don't need to check and cherry-pick beyond `SLE15SP2`.

### CODE REVIEW

**Review by tbazant — APPROVED** [review]
The changes look good!

File: xml/uefi-httpboot-server.xml [hunk 1]
```diff
@@ -215,72 +220,78 @@ subclass "HTTPClient" "HTTPClient:Arch:00016" {
...
+          <filename
       os="osuse">/srv/www/htdocs/opensuse/</filename><filename
       os="sles;sled">/srv/www/htdocs/sle/</filename>
-     directory. Edit then the
-     <filename
-      os="osuse">/srv/www/htdocs/opensuse/grub.cfg</filename><filename
-      os="sles;sled">/srv/www/htdocs/sle/grub.cfg</filename>
-     file. Use the following example as a reference:
-    </para>
+          directory. Edit then the
```
Comment (dariavladykina) [inline xml/uefi-httpboot-server.xml:289]:
```suggestion
          directory. Then edit the
```

File: xml/uefi-httpboot-server.xml [hunk 2]
```diff
@@ -385,65 +398,66 @@ $SERVER["socket"] == "[::]:443" {
...
+            Edit <filename>/etc/apache2/vhosts.d/vhost-ssl.conf</filename> to
+            change the private key and the certificate:
+          </para>
 <screen>SSLCertificateFile /etc/apache2/ssl.crt/server.crt
  SSLCertificateKeyFile /etc/apache2/ssl.key/server.key</screen>
-     <para>
-      Restart apache to activate the SSL support:
-     </para>
+          <para>
+            Restart apache to activate the SSL support:
```
Comment (dariavladykina) [inline xml/uefi-httpboot-server.xml:437]:
```suggestion
            Restart Apache to activate the SSL support:
```

File: xml/uefi-httpboot-server.xml [hunk 3]
```diff
@@ -5,100 +5,103 @@
...
-   replacement for PXE. HTTP Boot allows to boot a server from a URI over HTTP,
-   quickly transferring large files, such as the Linux kernel and root file
-   system from servers outside of your local network.
-  </para>
+    <para>
+      HTTP Boot combines DHCP, DNS and HTTP to make it possible to boot and
+      deploy systems over the network. HTTP Boot can be used as a
+      high-performance replacement for PXE. HTTP Boot allows to boot a server
+      from a URI over HTTP, quickly transferring large files, such as the Linux
+      kernel and root file system from servers outside of your local network.
```
Comment (dariavladykina) [inline xml/uefi-httpboot-server.xml:29]:
```suggestion
      kernel and root file system, from servers outside of your local network.
```

File: xml/uefi-httpboot-server.xml [hunk 4]
```diff
@@ -299,82 +310,84 @@ menuentry 'Installation IPv6' --class opensuse --class gnu-linux --class gnu --c
...
-     </para>
+        <sect4 xml:id="httpboot-enroll-certificate-physical-machine">
+          <title>Enroll the server certificate into the client firmware</title>
+          <para>
+            The exact procedure of enrolling the converted certificate depends
+            on the specific implementation of the client's firmware. For
+            certain hardware, you need to enroll the certificate manually via
+            the firmware UI using an external storage device with the
+            certificate on it. Machines with Redfish support can enroll the
+            certificate remotely. Consult documentation for your specific
```
Comment (dariavladykina) [inline xml/uefi-httpboot-server.xml:359]:
```suggestion
            certificate remotely. Consult the documentation for your specific
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some small corrections here. Thanks!

**Pull request merged**